modules/airgap/setup/get_artifacts.sh

```diff
@@ -39,21 +39,29 @@ get_assets() {
   elif [[ "$product" == "rke2" ]]; then
     url="https://github.com/rancher/rke2/releases/download/$version"
     wget "$url"/sha256sum-"$arch".txt
-    wget "$url"/rke2-images.linux-"$arch".txt
+    # Ref: https://docs.rke2.io/install/airgap
+    if [[ -n "$flags" ]] && [[ "$flags" =~ "cni" ]]; then
```

Comment:
sorry missed that earlier....can we have a doc here at the begining showing example of what are or could be those flags param?

Reply:
flags here referring to `server_flags`, I can rename it to be clear

Reply:
Updated